db/seeds.rb

```diff
@@ -1,3 +1,4 @@
+# frozen_string_literal: true
 # This file should contain all the record creation needed to seed the database with its default values.
 # The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
 #
```

Comment:
- [ ] Please create random messages in the seeds file to be pre-saved in the database for the controller to pick a random one of them. 🙏